assemblies/assembly-install-rhdh-osd-gcp.adoc

```diff
@@ -0,0 +1,15 @@
+[id="assembly-install-rhdh-osd-gcp"]
+= Installing {product} on {osd-short} on {gcp-brand-name}
+:context: assembly-install-rhdh-osd-gcp
+
+You can install {product-short} on {osd-short} on {gcp-brand-name} ({gcp-short}) using one of the following methods:
+
+* The {product} Operator
+* The {product} Helm chart
+
+// Operator procedure
+include::modules/installation/proc-install-rhdh-osd-gcp-operator.adoc[leveloffset=+1]
+
+// Helm procedure
+include::modules/installation/proc-install-rhdh-osd-gcp-helm.adoc[leveloffset=+1]
+
```

Comment:
As there is only one chapter in this book, this assembly would benefit from being just the introduction in master.adoc, and both procedures become chapter 1 and chapter 2.

Reply:
We can apply this to the other Install titles in a subsequent PR as well.